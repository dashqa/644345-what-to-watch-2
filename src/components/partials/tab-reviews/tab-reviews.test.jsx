import React from 'react';
import renderer from 'react-test-renderer';
import TabReviews from './tab-reviews';

const props = {
  dividedComments: {},
};

it(`Reviews tab correctly renders after relaunch`, () => {
  const tree = renderer.create(<TabReviews {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
