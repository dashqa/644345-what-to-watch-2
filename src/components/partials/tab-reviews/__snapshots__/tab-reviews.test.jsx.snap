// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews tab correctly renders after relaunch 1`] = `
<div
  className="movie-card__reviews movie-card__row"
/>
`;
